tests/integration/conftest.py

```diff
@@ -6,6 +6,10 @@
 import pytest
 
 
+BASE_PATH = str(Path(__file__).parent.absolute())
+REPO_BASE_PATH = BASE_PATH + "/../../"
```

Comment:
Why not use the Path object all the way through?

Reply:
`pexpect` needs a string as argument. I would also prefer a more pythonic way. But `pexpect` is not really pythonic.